### ISSUE TITLE
Proposal: create a verified `prettier` VS marketplace organization and transfer as `prettier.prettier-vscode` smoothly

### ISSUE THREAD

**Issue by JounQin** [prettier/prettier-vscode#3225]
It is possible to transfer an extension to another publisher org smoothly:

See also https://github.com/microsoft/vsmarketplace/issues/440

(Or just `prettier.prettier`)

**Comment by liuxingbaoyu** [comment]
![image](https://github.com/prettier/prettier-vscode/assets/30521560/2be04d98-292d-4339-a237-d141d8488d03)
Since this extension is already certified, I don't think it matters.

**Comment by ntotten** [comment]
Yeah, we would, but to get a verified organization you need to verify the domain (prettier.io) and the domain is owned by Meta, Inc. and nobody has been able to figure out how to update the DNS.
